code/core/src/core-server/utils/warnOnIncompatibleAddons.ts

```diff
@@ -2,7 +2,7 @@ import { logger } from '@storybook/core/node-logger';
 import {
   getIncompatibleStorybookPackages,
   getIncompatiblePackagesSummary,
-} from '../../../../lib/cli/src/doctor/getIncompatibleStorybookPackages';
+} from '../../../../lib/cli-storybook/src/doctor/getIncompatibleStorybookPackages';
```

Comment:
That doesn't seem correct. Should we ever import files from another package relatively?

Reply:
This relative import was added in:
https://github.com/storybookjs/storybook/pull/21883
reviewed by norbert, Michael and you ;) 

I would guess that @yannbf's purpose at the time was to avoid cyclical dependencies while still being able to share code. Using a relative import ensures that only that particular file gets pre-bundled without having to depend on the entire module.

But I have to say, it feels a bit odd, perhaps this file should just live in core-common? cc @ndelangen 
However, I would suggest doing that in another PR, and not in this one.

Reply:
It is technically definitely "correct".

TSup will bundle this import in. (and only this 1 bit you import and use).

But, I understand how this could be perceived as fragile or unwanted.
I'm fine with both.

If you're asking for my personal opinion:
Not everything needs to be a package.